src/App.js

```diff
@@ -1,5 +1,127 @@
+import {MissionUtils} from '@woowacourse/mission-utils';
+
 class App {
-  async play() {}
+  constructor() {
+    this.init();
+  }
+
+  init() {
+    MissionUtils.Console.print('숫자 야구 게임을 시작합니다.');
+    this.computerPickNum = this.randomPickNum();
+  }
+
+  randomPickNum() {
```

Comment:
```suggestion
  pickRandomNumbers() {
```
함수명은 수행하는 역할을 동사형으로 작성합니다.